app/actions/remote/groups.ts

```diff
@@ -54,22 +71,16 @@ export const fetchGroupsForChannel = async (serverUrl: string, channelId: string
 
 export const fetchGroupsForTeam = async (serverUrl: string, teamId: string, fetchOnly = false) => {
     try {
+        const {operator} = DatabaseManager.getServerDatabaseAndOperator(serverUrl);
         const client: Client = NetworkManager.getClient(serverUrl);
         const response = await client.getAllGroupsAssociatedToTeam(teamId);
 
         if (!fetchOnly) {
-            return await storeGroups(serverUrl, response.groups);
-        }
-
-        // return await storeGroups(serverUrl, response.groups, true);
-        const operator = DatabaseManager.serverDatabases[serverUrl]?.operator;
-        if (!operator) {
-            throw new Error(`${serverUrl} operator not found`);
+            return storeGroups(serverUrl, response.groups);
         }
 
-        return await prepareGroups(operator, response.groups);
+        return prepareGroups(operator, response.groups);
     } catch (error) {
-        forceLogoutIfNecessary(serverUrl, error as ClientErrorProps);
```

Comment:
why are you removing this here ?

Reply:
Whoops - this should not have been removed.